### DISCUSSION TITLE
License silero so we can train a custom model?

### DISCUSSION THREAD

**Discussion by sipvoip** [snakers4/silero-vad#246] [General]
Do you have the option of licensing so we can train our own model?

**Comment by snakers4** [comment]
Hi, no we do not provide model training options. If someone requires a custom model we do it commercially as a service.

**Comment by sipvoip** [comment]
Bummer ok, we are happy to pay,  but need something we can build and
control our own model. Thanks anyway, we will keep looting.

On Tue, Oct 18, 2022 at 8:26 PM Alexander Veysov ***@***.***>
wrote:

> Hi, no we do not provide model training options. If someone requires a
> custom model we do it commercially as a service.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/snakers4/silero-vad/discussions/246#discussioncomment-3910625>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAJRIERCKEMZZBOI4VBRYYLWD455BANCNFSM6AAAAAARIMGZUM>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
-- 

><>
nathan stratton

**Reply by snakers4** [reply]
For commercial inquiries please email or pm in telegram
